specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/LedgerDigestUpload.json

```diff
@@ -0,0 +1,331 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-02-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/ledgerDigestUpload/{ledgerName}": {
```

Comment:
>ledgerDigestUpload [](start = 142, length = 18)

Resource type should be plural, similar to databases or servers

Reply:
discussed via email, this will be changed in a different PR